internal/handlers/pg/msg_aggregate.go

```diff
@@ -86,6 +86,17 @@ func (h *Handler) MsgAggregate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 
 	username, _ := conninfo.Get(ctx).Auth()
 
+	v, _ := document.Get("maxTimeMS")
+	if v == nil {
+		v = int64(0)
+	}
+
+	maxTimeMS, err := commonparams.GetValidatedNumberParamWithMinValue(document.Command(), "maxTimeMS", v, 0)
+	if err != nil {
+		// unreachable for MongoDB GO driver, it validates maxTimeMS parameter
```

Comment:
I'm not sure if I understand that comment correctly, but if it says that this code is not reachable by tests, then this is already visible from them, and we have to way to update that comment when/if this line becomes reachable.

Reply:
So values that are negative and not int64 return driver error and doesn't call our backend. However, we wouldn't know when driver may change the behavior, if that happens failing gracefully (possibly wrong error code and message) is best we can do.

Meanwhile I could add tests for just there is an error without checking error type or content.

Reply:
Honestly not sure adding this test was good idea. Other PRs seems to remove driver tests so... We could easily revert https://github.com/FerretDB/FerretDB/pull/2929/commits/0945325285532113967c6b74d10df7024fa08076 if we want to.

Reply:
That commit link above tests `runCommand` already and it returns driver error :sweat_smile:

Reply:
Oh, I missed that 🙈